### PR TITLE
Precaching

### PR DESCRIPTION
**Pull request by halfak**
Implements a 'precached' utility that listens to RCStream and requests scores for revisions as they happen.

### CODE REVIEW

**Comment by he7d3r** [comment]
Looks good.

**Comment by halfak** [comment]
This pull request should wait for https://github.com/wiki-ai/ores/pull/57.  We'll need to make some updates to the way it reads the configuration once https://github.com/wiki-ai/ores/pull/57 gets merged.

File: ores/utilities/precached.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+                                             time.time() - start))
+
+            def on_connect(self):
+                logger.info("Connecting socketIO client to {0}."
+                            .format(stream_url))
+
+                # TODO: This should be limited to the wikis we actually care about
+                # but we identify a wiki by it's dbname and this pattern matching
+                # uses a domain name.
+                self.emit('subscribe', '*')  # Subscribes to all wikis
```
Comment (Ladsgroup) [inline ores/utilities/precached.py]: Are you precaching recent changes of **all** wikis? Is that really beneficial?

Comment (halfak) [inline ores/utilities/precached.py]: Does the comment above the emit describe the situation sufficiently?

**Pull request merged**